Chap_API_Struct.tex

```diff
@@ -2622,7 +2674,12 @@ \subsection{Size information attributes}
 
 %
 \declareNewAttribute{PMIX_NUM_NODES}{"pmix.num.nodes"}{uint32_t}{
-Number of nodes in this namespace.
+Number of nodes in this session or namespace.
+}
+
+%
+\declareNewAttribute{PMIX_NUM_SLOTS}{"pmix.num.slots"}{uint32_t}{
+Number of slots allocated to the session, namespace, or application.
```

Comment:
What is the difference between the number of slots and universe size?

Reply:
Added explanation